portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,4 +40,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(jsonMessages);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = request.getParameter("comment");
```

Comment:
Nit: consider inlining this. Doesn't seem to improve readability by being separate.

Reply:
What do you mean here?

Reply:
```
    messages.add(request.getParameter("comment"));
```